internal/service/cloudbackupsnapshotexportbucket/resource_cloud_backup_snapshot_export_bucket_test.go

```diff
@@ -69,6 +73,65 @@ func basicTestCase(tb testing.TB) *resource.TestCase {
 	}
 }
 
+func basicAzureTestCase(t *testing.T) *resource.TestCase {
+	t.Helper()
+
+	var (
+		projectID          = acc.ProjectIDExecution(t)
+		tenantID           = os.Getenv("AZURE_TENANT_ID")
+		bucketName         = os.Getenv("AZURE_BLOB_STORAGE_CONTAINER_NAME")
+		serviceURL         = os.Getenv("AZURE_SERVICE_URL")
+		atlasAzureAppID    = os.Getenv("AZURE_ATLAS_APP_ID")
+		servicePrincipalID = os.Getenv("AZURE_SERVICE_PRINCIPAL_ID")
+	)
+
+	return &resource.TestCase{
+		PreCheck: func() {
+			acc.PreCheckBasic(t)
+			acc.PreCheckCloudProviderAccessAzure(t)
+			acc.PreCheckAzureExportBucket(t)
+		},
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: configAzureBasic(projectID, atlasAzureAppID, servicePrincipalID, tenantID, bucketName, serviceURL),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "project_id", projectID),
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", bucketName),
+					resource.TestCheckResourceAttr(resourceName, "service_url", serviceURL),
+					resource.TestCheckResourceAttr(resourceName, "tenant_id", tenantID),
+					resource.TestCheckResourceAttr(resourceName, "cloud_provider", "AZURE"),
+					resource.TestCheckResourceAttrSet(resourceName, "role_id"),
+
+					resource.TestCheckResourceAttr(dataSourceName, "project_id", projectID),
+					resource.TestCheckResourceAttr(dataSourceName, "bucket_name", bucketName),
+					resource.TestCheckResourceAttr(dataSourceName, "service_url", serviceURL),
+					resource.TestCheckResourceAttr(dataSourceName, "tenant_id", tenantID),
+					resource.TestCheckResourceAttr(dataSourceName, "cloud_provider", "AZURE"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "role_id"),
+
+					resource.TestCheckResourceAttr(dataSourcePluralName, "project_id", projectID),
+					resource.TestCheckResourceAttr(dataSourcePluralName, "results.#", "1"),
+					resource.TestCheckResourceAttr(dataSourcePluralName, "results.0.bucket_name", bucketName),
+					resource.TestCheckResourceAttr(dataSourcePluralName, "results.0.service_url", serviceURL),
+					resource.TestCheckResourceAttr(dataSourcePluralName, "results.0.tenant_id", tenantID),
+					resource.TestCheckResourceAttr(dataSourcePluralName, "results.0.cloud_provider", "AZURE"),
+					resource.TestCheckResourceAttrSet(dataSourcePluralName, "results.0.role_id"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportStateIdFunc:       importStateIDFunc(resourceName),
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"container_id"},
```

Comment:
why is this ignored?

Reply:
Great catch! It wasn't needed, so I removed it